src/app/home/home.component.ts

```diff
@@ -16,18 +17,51 @@ export class HomeComponent implements OnInit {
 
   /** Username of authenticated user. */
   username: string;
+  /** Check if searched. */
+  isSeearched = false;
+  /** Activity Form. */
+  activityForm: any;
+  /** Search Text. */
+  searchText: any;
+  /** All User Activities. */
+  allActivities: any[] = [
+    { activity: 'client', path: '/clients'},
+    { activity: 'groups', path: '/groups'},
+    { activity: 'centers', path: '/centers'},
+    { activity: 'accounting', path: '/accounting'},
+    { activity: 'users', path: '/users'},
+    { activity: 'organization', path: '/organization'},
+    { activity: 'system', path: '/system'},
+    { activity: 'templates', path: '/templates'},
+    { activity: 'self-service users', path: '/self-service/users'},
+    { activity: 'self-service app-configuration', path: '/self-service/app-configuration'},
+    { activity: 'task management', path: '/self-service/task-management'}
```

Comment:
@BLasan add all routes as mentioned in ExpertSearchController.js file in community app.

Reply:
Sure. I'll update and send the PR soon.

Reply:
> @BLasan add all routes as mentioned in ExpertSearchController.js file in community app.

Is it okay to use all the paths mentioned in community-app?

Reply:
yes even if they don't work right now, I will check if some route change is required.

Reply:
Updated the activities mentioned in ExpertSearchController.js. Please review